int32 gripper
int32 vertical1
int32 vertical2
int32 vertical3
int32 horizontal1
int32 horizontal2
int32 horizontal3
int32 priority
int32 id
---
bool done
